addon/components/polaris-form.js

```diff
@@ -0,0 +1,132 @@
+import Component from '@ember/component';
+import layout from '../templates/components/polaris-form';
+import { normalizeAutoCompleteProperty } from '../utils/normalize-auto-complete';
+
+export default Component.extend({
+  tagName: 'form',
+  attributeBindings: [
+    'acceptCharset',
+    'action',
+    'autoCompleteInputs:autoComplete',
+    'encType',
+    'method',
+    'name',
+    'noValidate:novalidate',
+    'target',
+  ],
+  layout,
+
+  /**
+   * Space separated list of character encodings
+   *
+   * @type {String}
+   * @default null
+   * @public
+   */
+  acceptCharset: null,
+
+  /**
+   * Where to send form-data on submittal
+   *
+   * @type {String}
+   * @default null
+   * @public
+   */
+  action: null,
+
+  /**
+   * Grants the broswer the ability to autocomplete input elements
+   *
+   * @type {Boolean}
+   * @default null
+   * @public
+   */
+  autoComplete: null,
+
+  /**
+   * Media type when submiting content to server
+   *
+   * @type {String}
+   * @default null
+   * @public
+   */
+  encType: null,
+
+  /**
+   * Toggles if form submits on Enter keypress. Defaults to true.
```

Comment:
Isn't the statement about default a bit redundant?

Reply:
I know...just copy-pasta from Shopify...can tweak

Reply:
We've been keeping these as they came from Shopify for consistency with the React implementation, so let's carry on doing that if everyone's OK with doing so